docs/pt/docs/help-fastapi.md

```diff
@@ -0,0 +1,106 @@
+# Ajuda FastAPI - Obtendo Ajuda
+
+Você gosta do **FastAPI**?
+
+Gostaria de ajudar o FastAPI, outros usuários, e o autor?
+
+Ou gostaria de obter ajuda para utilizar o **FastAPI**?
+
+Há maneiras muito simples de ajudar (algumas envolvem somente um ou dois cliques). 
+
+Há várias maneiras de buscar ajuda também.
+
```

Comment:
Is there a particular reason why the fastapi Twitter handle (and the suggestion to follow) has been removed from here? Even though the tweets are probably in English (I'm guessing), Twitter itself has a built-in translator for the tweets... so even people who only speak Portuguese may end up being interested in following it.